### PR TITLE
Fix some typo and prevent misunderstanding

### PR DESCRIPTION
**Pull request by adev22**

### CODE REVIEW

File: A-command-line-args-flag.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Tabel berikut merupakan macam-macam fungsi flag yang tersedia untuk tiap jenis t
 
 Sebenarnya ada 2 cara deklarasi flag yang bisa digunakan, dan cara di atas merupakan cara pertama.
 
-Cara kedua mirip dengan cara pertama, perbedannya adalah kalau di cara pertama nilai pointer flag dikembalikan lalu ditampung variabel; sedangkan pada cara kedua, nilainya diambil lewat parameter pointer.
+Cara kedua mirip dengan cara pertama, perbedannya adalah kalau di cara pertama nilai pointer flag dikembalikan lalu ditampung variabel. sedangkan pada cara kedua, nilainya diambil lewat parameter pointer.
```
Comment (novalagung) [inline A-command-line-args-flag.md:111]: very quick small review. can you make the `sedangkan` capitalize? (to `Sedangkan`)

File: A-encoding-base64.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ var decodedString = string(decoded)
 fmt.Println(decodedString)
 ```
 
-Fungsi `base64.StdEncoding.EncodedLen(len(data))` menghasilkan informasi lebar data-ketika-sudah-di-encode. Nilai tersebut kemudian ditentukan sebagai lebar alokasi tipe `[]byte` pada variabel `encoded` yang nantinya digunakan untuk menampung hasil encoding.
+Fungsi `base64.StdEncoding.EncodedLen(len(data))` menghasilkan informasi lebar dari variable `data` ketika sudah di-encode. Nilai tersebut kemudian ditentukan sebagai lebar alokasi tipe `[]byte` pada variabel `encoded` yang nantinya digunakan untuk menampung hasil encoding.
```
Comment (novalagung) [inline A-encoding-base64.md:58]: And for this one, I think it sounds better with `... lebar variabel data ...`. so can you remote the `dari` word?

**Review by novalagung — CHANGES_REQUESTED** [review]
thank you for the contributions. just have two small reviews, other than that everything is perfect!

**Comment by adev22** [comment]
Changes Implemented

**Comment by novalagung** [comment]
added some additional changes from end

**Comment by novalagung** [comment]
ok, perfect, LGTM

**Comment by novalagung** [comment]
@adev22 thank you

**Pull request merged**